misc/userscripts/mac-1password-mini

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# This opens 1Password Mini with the domain of the current page filled in.
+# It does *not* provide auto-fill for the page.
+# It *only* works on macOS. (I don't know if there's an equivalent helper on
+# other platforms anyway)
+
+# Suggested mapping: `:bind <meta-\> spawn --userscript 1password-mini`
+
+domain=$QUTE_URL
+domain=${domain##*://} # remove protocol
+domain=${domain%%/*} # remove pathname
+domain=${domain%:*} # remove port
+domain=${domain##www.} # remove www. subdomain if present
+
+open x-onepassword-helper://search/${domain}
```

Comment:
Actually, it looks like [shellcheck is failing due to not wrapping this variable expansion in a quote](https://travis-ci.org/qutebrowser/qutebrowser/jobs/354437236), could you go ahead and do that please :)

Reply:
Ah indeed! Updated.